crates/biome_cli/tests/snapshots/main_commands_rage/with_formatter_configuration.snap

```diff
@@ -96,23 +96,23 @@ JavaScript Formatter:
   Bracket same line:            false
   Quote style:                  Double
   Indent style:                 Tab
-  Indent width:                 2
+  Indent width:                 IndentWidth(2)
   Line ending:                  Lf
   Line width:                   100
   Attribute position:           Auto
 
 JSON Formatter:
   Enabled:                      true
   Indent style:                 Space
-  Indent width:                 2
+  Indent width:                 IndentWidth(2)
   Line ending:                  Lf
   Line width:                   100
   Trailing Commas:              unset
 
 CSS Formatter:
   Enabled:                      false
   Indent style:                 Tab
-  Indent width:                 0
+  Indent width:                 IndentWidth(2)
   Line ending:                  Lf
   Line width:                   LineWidth(80)
```

Comment:
I think we want to display `2` and `80` to the users, instead of `IndentWidth(2)` and `LineWidth(80)`